### ISSUE TITLE
nvbuild.sh fails on jetpack 4.4 (L4T 32.4.3)

### ISSUE THREAD

**Issue by leonitousconforti** [mrcmunir/jetson_nano_overclock#1]
running sudo nvbuild.sh on Jetson-nano with jetpack 4.4 (L4T 32.4.3) produces an error at the end:

```
scripts/Makefile.build:649: recipe for target 'drivers/video/tegra' failed
make[2]: *** [drivers/video/tegra] Error 2
```

the root cause seems to come earlier:
```
  CC      drivers/usb/gadget/udc/core.o
  CC      drivers/usb/gadget/udc/tegra_usb_cd_ops.o
  CC      drivers/video/tegra/host/debug.o
xxd -i /jetson_nano_overclock/kernel/nvidia/drivers/video/tegra/host/nvhost_events.json drivers/video/tegra/host/nvhost_events_json.h
/jetson_nano_overclock/kernel/nvidia/drivers/video/tegra/host/Makefile:66: recipe for target 'drivers/video/tegra/host/nvhost_events_json.h' failed
make[4]: *** [drivers/video/tegra/host/nvhost_events_json.h] Error 127
scripts/Makefile.build:649: recipe for target 'drivers/video/tegra/host' failed
make[3]: *** [drivers/video/tegra/host] Error 2
make[3]: *** Waiting for unfinished jobs....
  CC      drivers/usb/gadget/udc/tegra_usb_cd.o
  CC      drivers/usb/misc/usb3503.o
  LD      drivers/usb/misc/built-in.o
```

If it makes a difference, I am running this on a minimized system because I did the initial setup in headless mode and have only ever logged in via ssh
```
Welcome to Ubuntu 18.04.5 LTS (GNU/Linux 4.9.140-tegra aarch64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
This system has been minimized by removing packages and content that are
not required on a system that users do not log into.

To restore this content, you can run the 'unminimize' command.
```

**Comment by leonitousconforti** [comment]
The same problem occurs on both the master and the overclock-extreme branch, and running the script with sudo doesn't change anything

**Comment by mrcmunir** [comment]
You try make clean on kernel-4.9 folder and running again nvbuild.sh will be resolved the make 127 error.

**Comment by leonitousconforti** [comment]
I just tried running make clean on kernel-4.9 folder and then running nvbuild.sh again but the same error happened

**Comment by leonitousconforti** [comment]
The error has been fixed. I mentioned before that I was running a minimized system so some of the packages for the gui and other non essential packages were not install on setup.

The Make error 127 was because the xxd command was not found. All I had to do was install the xxd package:
```
sudo apt-get install xxd
```

**Comment by mrcmunir** [comment]
@leonitousconforti Glad you fixed it
I did not know where the problem could be beyond a possible cache that is the most common problem since it worked for me 
another recommended option is to use crosscompiler with PC x86-64 host .
https://developer.ridgerun.com/wiki/index.php?title=Jetson_Nano/Development/Building_the_Kernel_from_Source

**Comment by leonitousconforti** [comment]
When I said I had it working, I had got to the point where I generated the Image file in the kernel/kernel-4.9/arch/arm64/boot/ directory - which I thought was all the way. Unfortunately, I can't seem to get the system to recognize the new kernel image. These are the step I did:

1. ```sudo ./nvbuild.sh```
2. inside of kernel/kernel-4.9 ```sudo make modules && sudo make modules_install```
3. inside of kernel/kernel-4.9 ```sudo make && sudo make install```

After that, I find the new image generated at kernel/kernel-4.9/arch/arm64/boot/Image. I have tried copying that file to /boot/Image
4. ```sudo cp kernel/kernel-4.9/arch/arm64/boot/Image /boot/Image```
but when I reboot the board and run jetson_clocks, the cpu still only runs at 1.5GHz

Am I missing something in my steps? or is there some other step I need to perform to enable the cpu to run at 2GHz?
